sdk/core/System.BinaryData/src/System.BinaryData.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>Experimental types that might eventually move to Azure.Core</Description>
+    <AssemblyTitle>Microsoft Azure Client Pipeline Experimental Extensions</AssemblyTitle>
+    <Version>1.0.0</Version>
+    <PackageTags>Microsoft Azure Client Pipeline</PackageTags>
+    <Nullable>enable</Nullable>
+    <TargetFrameworks>net461;netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
Why these target frameworks?

Reply:
I copied what was used here - https://github.com/dotnet/runtime/compare/master...ericstj:binaryData?expand=1

Should I use RequiredTargetFrameworks instead?

Reply:
I would just set this to RequiredTargetFrameworks which is only netstandard2.0 at this point. Based on the spec it seems like ns2.0 is the target framework they want.

Reply:
I don't think we should spend time trying to mimic the exact dotnet/runtime build configuration.

Reply:
> I don't think we should spend time trying to mimic the exact dotnet/runtime build configuration.

Why's that, this is intentional and the way we will ship the package out of dotnet/runtime.  It shouldn't require you to spend any time.  We intentionally cross target in order to get the minimal dependencies.  We have lots of feedback that desktop customers don't want  a single library to opt them into carrying the netstandard2.x facades on desktop, thus the `net461` TargetFramework.  We also don't want folks to carry a System.Memory package reference on frameworks that don't need it, thus the `netstandard2.1` TargetFramework.

Reply:
Are we planning to ship this package out of `azure-sdk-for-net` repo?

Reply:
So if the code would move anyway and would benefit from the existing `dotnet/runtime` infrastructure why spend time trying to reproduce it here.

Reply:
The first version of this package is needed to ship out of here.  It will not ship out of dotnet/runtime until 6.0.

Reply:
Understood, I had outdated information.

Reply:
@ericstj Is a netstandard2.1 TFM justified here?  I understand the net461 one, to allow the package reference set to trim out the .NET Standard -> .NET Framework plumbing; but what's the gain from a ns2.1 version?

Reply:
NS2.1 was to avoid a PackageReference to System.Memory, which you can see here https://github.com/dotnet/runtime/compare/master...ericstj:binaryData?expand=1#diff-33a97179e044ae2f2d1abd0ca71fdfd37bde388e2abc300fee3c8fabba57fe34R8-R11

I guess here they're getting it transitively from System.Text.Json... I think it can be removed from both in that case, since it doesn't actually reduce the package or assembly graph needed by this lib.

Reply:
So shall I remove 2.1?

Reply:
Removed 2.1